Makefile

```diff
@@ -46,7 +46,7 @@ GO111MODULE = on
 # ====================================================================================
 # Setup Kubernetes tools
 
-UP_VERSION = v0.13.0
+UP_VERSION = v0.16.1
```

Comment:
Are these changes required for this PR? If not, please remove them. 

Same goes for the `build` submodule.

Reply:
I was actually completely unable to get the build process to work without this change. I think I saw some other traffic about it at the time.